src/components/DatePicker/DatePicker.vue

```diff
@@ -19,15 +19,15 @@
                       autocomplete="chrome-off"
                       :error-messages="errorMsg"
                       :error="!!errorMsg"
-                      :value="dateText"
+                      :value="displayDate"
                       :label="title"
                       :name="Math.random()"
                       :rules="inputRules"
                       :disabled="disablePicker"
-                      v-on="on"
                       v-on:keydown="$event.preventDefault()"
                       v-on:keyup.enter="emitDate(dateText)"
                       v-on:click:append="on.click"
+                      v-on="on"
```

Comment:
All these v-ons look funny. How about using the shorthand for events?

https://vuejs.org/v2/guide/syntax.html#v-on-Shorthand

Reply:
Will try!

Reply:
Updated to short hand events